dynamiq/utils/embedding_validators.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any
+
+
+def validate_embedding(embedding: Any) -> tuple[bool, str]:
+    """
+    Validates that an embedding is not empty.
+
+    Args:
+        embedding: The embedding vector to validate
+
+    Returns:
+        Tuple containing:
+            - Boolean indicating if embedding is valid
+            - String containing error message if invalid, empty string if valid
+    """
+    try:
+        if embedding is None:
+            return False, "Embedding is None"
```

Comment:
Please use `raise SomeCustomError("error msg")` in case of error instead of returning a tuple of (bool, error_msg). So after handle validation by try..except